README.md

```diff
@@ -49,6 +49,7 @@ See [the examples](./examples/README.md) for more detailed usage or read about a
 * `failureThreshold`: (default `0`) Sets the threshold that would trigger a test failure based on the `failureThresholdType` selected. This is different to the `customDiffConfig.threshold` above, that is the per pixel failure threshold, this is the failure threshold for the entire comparison.
 * `failureThresholdType`: (default `pixel`) (options `percent` or `pixel`) Sets the type of threshold that would trigger a failure.
 * `updatePassedSnapshot`: (default `false`) Updates a snapshot even if it passed the threshold against the existing one.
+* `blur`: (default `0`) Applies Gaussian Blur on generated images. Useful when you have noise after scaling images per different resolutions on your target website.
```

Comment:
Is this out of 100? Can you document that please?

Reply:
From your description: `Applying a blur, usually 1-2 radius, removes this noise and fixes this issue`

I think adding something about that would be useful in the README.

Reply:
Sure will update description